includes/Media.php

```diff
@@ -302,7 +335,7 @@ public static function rest_api_init() {
 	 *
 	 * @return array $response;
 	 */
-	public static function wp_prepare_attachment_for_js( $response, $attachment ) {
+	public function wp_prepare_attachment_for_js( $response, $attachment ) {
```

Comment:
`$image = self::get_thumbnail_data( $thumbnail_id );` below can be replaced with `$image = $this->get_thumbnail_data( $thumbnail_id );`